docs/contribute/create-diagram.md

```diff
@@ -0,0 +1,111 @@
+    - Hold down ++opt++ and drag to duplicate an element.
+
+### Style
+
+- Use the premade diagram assets as starting points.
+  By default, you can adjust the width of the premade labels, but the height is automatically sized to the number of
+  lines of text.
+  To freely customize a component, right-click on it, **detach instance**, and **remove auto layout**.
+- Use the pre-defined [color styles](https://help.figma.com/hc/en-us/articles/360039820134-Manage-and-share-styles) or
+  black (#00000).
+- Use rounded corners of radius 2 for rectangular labels and containers.
+- Evenly [align](https://help.figma.com/hc/en-us/articles/360039956914-Adjust-alignment-rotation-and-position) elements.
+- Represent similar conceptual elements using similar styles.
+  For example, represent two nodes using a dark gradient, and represent two external components using a light gradient.
+- You can reuse existing icons from any diagram on any page.
+  For example, there are already icons to represent databases, dapps, keys, locks, and logos.
+
+!!! tip
+
+    Hold down ++shift++ when drawing, resizing, and rotating to create perfect horizontal and vertical lines.
+
+#### Text
+
+- Use font **Roboto** for all text.
+- Use font sizes between 10-18.
+- Use [sentence-style capitalization](https://docs.microsoft.com/en-us/style-guide/capitalization) in labels and titles.
+
+#### Arrows and lines
+
+- Use a thickness of 2 for arrows, lines, borders, and other strokes.
+- Use **Triangle** arrow heads.
+- Use straight arrows and lines, with right-angle bends if needed.
+  Don't use diagonal arrows and lines.
+  If possible, don't overlap arrows and lines.
+  To create additional bends in an arrow or line, **detach instance** (if applicable), double-click the arrow or line,
+  and click and drag the anchor points.
+- Leave about 3px of space between arrow heads and the elements they point to.
+  Line ends without arrow heads should touch the connecting element.
+
+!!! example
+
+    ![Example diagram](../assets/images/besu-tessera-high-availability.png)
+
+See the
+[Figma documentation on the Arrow Tool](https://help.figma.com/hc/en-us/articles/360040450133-Using-Shape-Tools#h_677f8eba-73c4-4987-a64b-c0226aaec392)
+for more information.
+
+### Export
+
+To export your diagram:
+
+1. Select the frame of your diagram.
+    Make sure all elements of your diagram are contained in the frame.
+    The name of this frame will be the name of the exported image.
+
+1. Scroll to the bottom of the right sidebar.
+    In the **Export** section, choose **2x** scale (for retina screens) and **PNG** or **JPG** file format.
+
+1. Export the diagram to the image folder of the documentation site (for example, `doc.goquorum/docs/images`).
+
+See [Figma's guide to exports](https://help.figma.com/hc/en-us/articles/360040028114-Guide-to-exports-in-Figma) for more
+information.
+
+Embed the diagram into a documentation page using `![](<path to your image file>)`, then create your PR.
```

Comment:
```suggestion
Embed the diagram into a documentation page using `![<alternative description or your image>](<path to your image file>)`, then create your PR.
```

it's important to add an "alt" text in the `[ ]`part of the image code so that it provides alternative content for screen readers (accessibility) and search engines. See [Markdown doc](https://daringfireball.net/projects/markdown/syntax#img) and [more explanation on why alt text is useful](https://moz.com/learn/seo/alt-text).

If you want, you can also add a title to your image that will display when you over the image with the mouse.
for that, write your image tag as `![<alternative description or your image>](<path to your image file> "my image title")`.
Note the `"` around the title, they are important.
Titles are not mandatory like alt text. However, it is convenient and helps the user as much as contributors to know what is behind an image and also have the image keywords available for search (otherwise site search doesn't know what is in your images).
Consider the title as the legend of image and the alt as a full description for people using screen readers.
At some point we could decide to display the legend of the image bellow the image using this title if most images have one.